### PR TITLE
[OpenShift] Add triggers defaults in CR instead of transformers

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

Triggers now support `config-default-triggers` configmap as part of this PR https://github.com/tektoncd/triggers/pull/1182.
PR introduced new attribute `trigger` in TektonConfig so that user can change the value of service account using CR itself.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: pkg/reconciler/kubernetes/tektontrigger/tektontrigger.go [hunk 1]
```diff
@@ -134,14 +139,16 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tt *v1alpha1.TektonTrigg
 // transform mutates the passed manifest to one with common, component
 // and platform transformations applied
 func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp v1alpha1.TektonComponent) error {
-	instance := comp.(*v1alpha1.TektonTrigger)
+	trigger := comp.(*v1alpha1.TektonTrigger)
 	triggerImages := common.ToLowerCaseKeys(common.ImagesFromEnv(common.TriggersImagePrefix))
+	instance := comp.(*v1alpha1.TektonTrigger)
```
Comment (sm43) [inline pkg/reconciler/kubernetes/tektontrigger/tektontrigger.go:144]: we have 2 `TektonTrigger` obj here `trigger` and `instance` do we need to have both?

Comment (savitaashture) [reply pkg/reconciler/kubernetes/tektontrigger/tektontrigger.go]: No we don't need 2 obj
Updated :+1:

**Review by vdemeester — APPROVED** [review]

**Pull request merged**